pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -29,6 +30,27 @@ func (e *CloudWatchExecutor) executeLogActions(ctx context.Context, queryContext
 				return err
 			}
 
+			if query.Model.Get("groupResults").MustBool() && len(dataframe.Fields) > 0 {
```

Comment:
Please add a comment here explaining that the cloudwatch api response isn't grouped automatically when used `stats ... by ...`